doc/tutorial/introduction/introduction_align.cpp

```diff
@@ -41,11 +41,11 @@ int main()
 //! [alignment]
     // Call a pairwise alignment with edit distance and traceback.
     for (auto && res : align_pairwise(std::make_pair(sequences[0], sequences[1]),
-                                      align_cfg::edit | align_cfg::result{align_cfg::with_trace}))
+                                      align_cfg::edit | align_cfg::result{with_alignment}))
     {
         // Print the resulting score and the alignment.
-        debug_stream << res.get_score() << '\n';              // => -4
-        debug_stream << res.get_alignment() << '\n';          // =>       0     .    :
+        debug_stream << res.score() << '\n';              // => -4
+        debug_stream << res.alignment() << '\n';          // =>       0     .    :
```

Comment:
Please indent the comment according to the following lines. Otherwise you destroy the expected output.

Reply:
Yes, I have seen this also, but was to lazy to adapt it right away 😄 👍